lib/state.js

```diff
@@ -95,6 +95,10 @@ StateResolver.prototype.capture = function() {
   // Gather the stack frames first
   var that = this;
   var frames = that.resolveFrames_();
+  this.nativeErrorMirror =
+    this.state_.frame(0).evaluate('\'Cannot display native properties.\'');
+  this.getterErrorMirror =
+    this.state_.frame(0).evaluate('\'Cannot display properties with getters.\'');
```

Comment:
I wasn't aware of this channel for communicating errors. This is much better than a string value.